app/graphql/mutations/destroy/base.rb

```diff
@@ -0,0 +1,14 @@
+module Mutations
+  module Destroy
+    class Base < ::Mutations::Base
+      def update_status_generic(record, args, ctx)
+        record.update(status: 1)
+        save_attributes record, args, ctx, type: :update?
+      end
+
+      def destroy_generic(record, args, ctx)
+        save_attributes record, args, ctx, type: :update?
+      end
```

Comment:
Couple things here:

1) I think the update status is specific enough to the one user destroy action that we can just put that in the user destroy mutation directly.
2) I think your copy paste here needs updating. The `save_attributes` doesn't handle destroying objects.